api/src/main/java/org/grammatek/apis/UsersApi.kt

```diff
@@ -57,11 +63,11 @@ class DevelopersApi(basePath: kotlin.String = org.grammatek.apis.DevelopersApi.C
     */
     @Suppress("UNCHECKED_CAST")
     @Throws(IllegalStateException::class, IOException::class, UnsupportedOperationException::class, ClientException::class, ServerException::class)
-    fun correctApiPost(text: kotlin.String) : YfirlesturResponse {
-        val localVarResponse = correctApiPostWithHttpInfo(text = text)
+    fun correctApiPost(correctRequest: CorrectRequest) : CorrectResponse {
```

Comment:
if the above class would be named `CorrectApi`, could we name this call then `postCorrect` ?

Reply:
I will look into how we can change that, I believe it is generated from the path of the API which is /correct.api

Reply:
Hmm, no we shouldn't have to change anything after it's been generated, so if this is a convetion derived from the path, then either we change the path or leave it as is and just live with the naming.

Reply:
Maybe this is something we should revisit once we have our own service up then?

Reply:
yes, leave these methods as is, just change the API from UserApi to CorrectApi.